src/CH1_T1/main.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+
+float convert_from_fahrenheits_to_celsius( const float t ) 
+{
+    return (t - 32) *  5 / 9;
+}
+
+int main (int argc, char** argv)
+{
+    if(argc < 1) {
+        printf("Error reading the cmd filename\n\n");
+    } else {
+        printf("Name of binary file is %s\n\n", argv[0]);
```

Comment:
This task (Write a program which prints a name of its binary file) should be in a separate file (program).